BigAssApi.js

```diff
@@ -196,31 +216,35 @@ function BigAssFan (name, id, address, master) {
 
     this.fan = new BigAssProperty('fan', this);
     this.fan.createGetField('isOn', ['FAN', 'PWR'], true, undefined, "ON", "OFF");
-    this.fan.createGetField('speed', ['FAN', 'SPD'], true, 'CURR');
+    this.fan.createGetField('speed', ['FAN', 'SPD'], true, 'ACTUAL'); //0-7
```

Comment:
Very true, or I might have confused this, could there be both a current and actual?